yarn/assets/configuration/spec.yaml

```diff
@@ -68,6 +68,13 @@ files:
             value:
               example: true
               type: boolean
+          - name: collect_apps_all_states
+            description: |
+              Enable this parameter when using `collect_app_metrics` to collect metrics from applications of all states.
+              For example, collect metrics from applications with 'PENDING', 'FINISHED', and other states.
+            value:
+              example: false
+              type: boolean
```

Comment:
We could also have a list of states to allow customers to only fetch some specific states, and set the default value to a list only containing the `running` state. It would be a little bit more flexible. What do you think?

Reply:
That's a good idea! I'll add that